File: C/Chapter 3 - Control Statement/2K221008/patteren_star_pyramid_descending.c
//Loops:

//Pattern Programs

/*

Ex. Star pyramid Pattern - Descending

* * * * *
 * * * *
  * * *
   * *
    *

Here, <blankspace> is denoted as #

*/

#include<stdio.h>
void main()
{
    int count,i,j;

    printf("Enter the count: ");
    scanf("%d",&count);
    printf("\n");

    for(i=1;i<=count;i++){
        for(j=1;j<i;j++){
            printf(" ");
        }
        for(j=i;j<=count;j++){
            printf("* ");
        }
        printf("\n");
    }
}

File: Back End/Python/Chapter 4 - Control Statements/2K221229/patteren_star_pyramid_descending.py
# //Loops:
#
# //Pattern Programs
#
# /*
#
# Ex. Star pyramid Pattern - Descending
#
# * * * * *
#  * * * *
#   * * *
#    * *
#     *
#
# Here, <blankspace> is denoted as #
#
# */

count = int(input("Enter the count: "))

for i in range(0, count + 1, 1):
    for j in range(1, i + 1, 1):
        print(" ", end='')
    for j in range(i, count + 1, 1):
        print("* ", end='')
    print(end='\n')
